### DISCUSSION TITLE
People Driven Edition

### DISCUSSION THREAD

**Discussion by okuramasafumi** [okuramasafumi/alba#336] [Feedback]
Welcome to the new release of Alba! There are two new features and one bugfix.

- `helper` to share behaviors with inline associations
- `prefer_resource_method!` to configure Alba to use resource method first with `attributes`
- Fixing resource name inference bug

This released is named as "People Driven Edition" because all these changes are something that I haven't noticed we need. Thank you [@george-ayris](https://github.com/george-ayris), [@danielmalaton](https://github.com/danielmalaton), [@datbth](https://github.com/datbth) and [@kapso](https://github.com/kapso) to help making it better!

One important note, this might be the last release to support Ruby 2! It's already EOL so I'd like to encourage people to use Ruby 3.

<hr /><em>This discussion was created from the release <a href='https://github.com/okuramasafumi/alba/releases/tag/v2.4.0'>People Driven Edition</a>.</em>

**Comment by kapso** [comment]
@okuramasafumi thanks for the mention, we are in the process of migrating a large application from `jsonapi-serializer` to Alba, and so far its going well. We were not happy with JSON:API format, and were looking for something that is both feature rich, and performant, and I think Alba fits in that category really well. Thanks a lot for your work on this gem.
